CHANGELOG.md

```diff
@@ -439,10 +439,10 @@ type ChainAssetMap<T> = {
 - `SwapSDK.prototype.getRequiredBlockConfirmations` now has a return type of
   `ChainMap<number | null>` instead of `ChainMap<number | undefined>` to be more
   consistent with existing return types.
-- `SwapSDK.prototype.getAssets` now uses the `cf_supported_assets` RPC method to
+- `SwapSDK.prototype.getAssets` now uses the `cf_available_pools` RPC method to
   determine the supported assets of the connected Chainflip network. The format
   of the returned data was not changed.
-- `SwapSDK.prototype.getChains` now uses the `cf_supported_assets` RPC method to
+- `SwapSDK.prototype.getChains` now uses the `cf_available_pools` RPC method to
```

Comment:
was this maybe changed by accident?

Reply:
no, but it's a backend only change

Reply:
hmm okay, because it falls under 1.3.0 changes

Reply:
tru, will revert this, thought it was part of the 1.9 changelog

Reply:
thx